### ISSUE TITLE
Cannot drop diagram onto Camunda Modeler anymore

### ISSUE THREAD

**Issue by nikku** [camunda/camunda-modeler#4483]
### Describe the bug

When I try to drop a diagram onto the modeler then `path` as a property is no longer provided. This means that our drop logic blows up with the following error:

```
Cannot read properties of undefined (reading 'split')
    at getFileTypeFromExtension (webpack://camunda-modeler-client/src/app/App.js:2344:0)
    at readFileFromPath (webpack://camunda-modeler-client/src/app/App.js:726:0)
    at Array.map (<anonymous>)
    at App.readFileList (webpack://camunda-modeler-client/src/app/App.js:709:0)
    at Object.handleDrop (webpack://camunda-modeler-client/src/app/App.js:1974:0)
    at handleDrop (webpack://camunda-modeler-client/src/app/drop-zone/DropZone.js:78:0)
```

### Steps to reproduce

1. Open Modeler
2. Drop file from local file-system
3. See error pop up

### Expected behavior

I can drop (and open) files from the local file system.

### Environment

- OS: Linux (Ubuntu)
- Camunda Modeler Version: v5.26.0
- Execution Platform: Any
- Installed plug-ins: None

Works in Camunda Modeler v5.25.0.

### Additional context

Capture visualizing the issue:

![capture DwZ3PQ_optimized](https://github.com/user-attachments/assets/39177455-1c45-46e8-8bdf-7b8c3db44afb)

**Comment by barmac** [comment]
I can reproduce it on nightly (MacOS, dropping a file from the Desktop)
![image](https://github.com/user-attachments/assets/671e2483-0e20-423b-b636-8aa25473187b)

**Comment by barmac** [comment]
file.path is not attached:
![image](https://github.com/user-attachments/assets/57b90be0-8ddf-4072-a8aa-4c1bdde71beb)

Now it's time to read the Electron breaking changes :D

**Comment by barmac** [comment]
Yes, it is an Electron BC:

> Removed: File.path
> The nonstandard path property of the Web File object was added in an early version of Electron as a convenience method for working with native files when doing everything in the renderer was more common. However, it represents a deviation from the standard and poses a minor security risk as well, so beginning in Electron 32.0 it has been removed in favor of the [webUtils.getPathForFile](https://www.electronjs.org/docs/latest/api/web-utils#webutilsgetpathforfilefile) method.

https://www.electronjs.org/docs/latest/breaking-changes#removed-filepath

**Comment by barmac** [comment]
I played with this a bit, and it requires us to modify the preload script. This will then introduce a requirement on the file drop handler (dropzone) to also connect to the preload script.
tldr: I suggest we downgrade Electron for now, and resolve the issue for the next release under assumption that the "broken dropdown issue" is not breaking due to downgrade.

**Comment by barmac** [comment]
I am checking v31.4.0 right now.

**Comment by barmac** [comment]
Result: https://github.com/camunda/camunda-modeler/pull/4485

**Comment by barmac** [comment]
When we upgrade to v32, we need to adjust to the breaking change.

**Comment by barmac** [comment]
Fortunately, in the app we don't usually deal with web File objects, so that should ease the migration. We can still carry `file.path` around as a plain javascript object property like we do in Tabs.
